STM32F405RGTx/Core/Src/main.cpp

```diff
@@ -748,6 +790,8 @@ void HAL_TIM_PeriodElapsedCallback(TIM_HandleTypeDef *htim)
   if (htim->Instance == TIM9) {
     HAL_IncTick();
   }
+
+  CANBus_timer_isr(htim);

```

Comment:
Can you put this inside the `/* USER CODE BEGIN Callback */`?

Also, I don't think `CANBus_timer_isr` is the best name for this, since it doesn't really have much to do with the bus itself. Something like `WLoopCAN_timer_isr` would be more appropriate in my opinion.